app/js/checkAnswers.js

```diff
@@ -1,28 +1,33 @@
 const questionAmount = 30// amount of questions
 
-document.querySelector('#finish').addEventListener('click', function(e) {
-    var unanswered = questionAnswered()
-    if (unanswered == false) {
+document.querySelector('#finish').addEventListener('click', function() {
+    let unanswered = questionAnswered()
+    if (unanswered == false && !Object.values(flaggedQuestions).includes(true)) {
         showResults()
-    } else {
-        document.getElementById('modal-title').textContent = `You have ${unanswered.length} unanswered questions.`
+    }
+    else {
+        let flagList = document.getElementById('flag-list')
+        let unansweredList = document.getElementById('unanswered-list')
+        document.getElementById('modal-title').textContent = 'You have ' + unanswered.length + ' unanswered questions.'
```

Comment:
why have you stopped using a template string here? that was better!

Reply:
Changed it back!